schemas/data/messageClicks.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/metrics/direct-marketing/message-clicks",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Message Clicks",
```

Comment:
I am not sure if we really need a third metric after `web/linkclicks` and `advertising/clicks` already. While it may make sense for each paradigm to have their own metric for their click activities, I guess all the three are essentially represent weblink clicks.

Reply:
I could see this either way.  Agreed they are all just clicks on a link.  But the context of each is very different, and therefore the meaning or intent of the click is very different.  On the other hand, we capture all that context, so a processor could always sort it out later.  @cluby-adobe can you weigh in on this?